benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/BenchmarksSave.kt

```diff
@@ -36,7 +36,15 @@ fun saveBenchmarkStatsOnDisk(name: String, stats: BenchmarkStats) {
 
         SystemFileSystem.createDirectories(path.parent!!)
         SystemFileSystem.sink(path).writeText(text)
-        println("Results saved to ${SystemFileSystem.resolve(path)}")
+        println("CSV results saved to ${SystemFileSystem.resolve(path)}")
+        println()
+
+        val jsonString = stats.toJsonString()
```

Comment:
I would introduce a separate run argument for json writing (saveStatsAsJson, renaming the previous one to saveStatsToCSV?)

Reply:
Done. Now it has 2 arguments `saveStatsToJSON` and `saveStatsToCSV`